README.md

```diff
@@ -2,32 +2,68 @@ Upstream status (`master` branch):
 
-Links:
-* Official github code repository: [offlineimap]
-* Website: [website]
-* Wiki: [wiki]
-* Blog: [blog]
+[![Python versions supported](https://img.shields.io/pypi/pyversions/offlineimap.svg)](https://pypi.python.org/pypi/offlineimap)
+[![Package stability](https://img.shields.io/pypi/status/offlineimap.svg)](https://pypi.python.org/pypi/offlineimap)
+[![PyPI status](https://badge.fury.io/py/offlineimap.svg)](https://pypi.python.org/pypi/offlineimap)
+[![Kit format](https://img.shields.io/pypi/format/offlineimap.svg)](https://pypi.python.org/pypi/offlineimap)
+[![Implementation](https://img.shields.io/pypi/implementation/offlineimap.svg)](https://pypi.python.org/pypi/offlineimap)
 
-# OfflineIMAP
+[![GitHub last commit](https://img.shields.io/github/last-commit/OfflineIMAP/offlineimap.svg)](https://github.com/OfflineIMAP/offlineimap/commits/master)
+[![Github Tag](https://img.shields.io/github/tag/OfflineIMAP/offlineimap.svg)](https://github.com/OfflineIMAP/offlineimap/releases)
+[![Follow on twitter](https://img.shields.io/twitter/follow/OfflineIMAP.svg?style=social&logo=twitter)](https://twitter.com/intent/follow?screen_name=OfflineIMAP)
 
-***"Get the emails where you need them."***
+[offlineimap]: https://github.com/OfflineIMAP/offlineimap
+[website]: http://www.offlineimap.org
+[wiki]: https://github.com/OfflineIMAP/offlineimap/wiki
+[blog]: http://www.offlineimap.org/posts.html
+[docs]: https://offlineimap.readthedocs.io/
+[mailing_list]: http://lists.alioth.debian.org/mailman/listinfo/offlineimap-project
+[mailing_list_archive]: http://dir.gmane.org/gmane.mail.imap.offlineimap.general
+[code_of_conduct]: https://github.com/OfflineIMAP/offlineimap/blob/master/CODE_OF_CONDUCT.md
+[copying_license]: https://github.com/OfflineIMAP/offlineimap/blob/master/COPYING
+[changelog]: https://github.com/OfflineIMAP/offlineimap/blob/master/Changelog.md
+[maintainers]: https://github.com/OfflineIMAP/offlineimap/blob/master/MAINTAINERS.rst
+[manifest]: https://github.com/OfflineIMAP/offlineimap/blob/master/MANIFEST.in
+[todo]: https://github.com/OfflineIMAP/offlineimap/blob/master/TODO.rst
+[conf]: https://github.com/OfflineIMAP/offlineimap/blob/master/offlineimap.conf
+[conf_minimal]: https://github.com/OfflineIMAP/offlineimap/blob/master/offlineimap.conf.minimal
+[issues_new]: https://github.com/OfflineIMAP/offlineimap/issues/new
+[pull_request_new]: https://github.com/OfflineIMAP/offlineimap/compare
+[issues_need_contrib]: https://github.com/OfflineIMAP/offlineimap/issues?q=is%3Aopen+is%3Aissue+label%3A"need+contributor!"
+[homebrew_mac_install]: https://brewinstall.org/Install-offlineimap-on-Mac-with-Brew/
+[macports_mac_install]: https://www.macports.org/ports.php?by=name&substr=offlineimap
+
+<p style="center">
+<h1><img src="https://upload.wikimedia.org/wikipedia/commons/1/13/OfflineIMAP_logo.png" alt="OfflineIMAP"/>OfflineIMAP</h1>
```

Comment:
Remove this link, please.

Reply:
Which link? The macports link? The macports link is used later, in the section "How to download". Don't we want Mac users who using MacPorts, to be able to directly go to the macports page and learn how to obtain the software?

Reply:
The one I comment: wikimedia.

Reply:
That's the graphical logo icon for the software.  Shouldn't we display this?

Reply:
It's outdated. Use this one http://www.offlineimap.org/assets/logo/offlineimap-scalable.svg or anyone in https://github.com/OfflineIMAP/offlineimap.github.io/tree/master/assets/logo